OSDOCS-10115: Remove contractions (#73996)
openshift/openshift-docs: modules/ossm-load-test-results.adoc

The Envoy proxy collects raw telemetry data after a response is sent to the client.
The time spent collecting raw telemetry for a request does not contribute to the total time taken to complete that request.

> However, since the worker is busy handling the request, the worker will not start handling the next request immediately.

Reviewer: Per ISG s/since/because
```suggestion
However, because the worker is busy handling the request, the worker does not start handling the next request immediately.
```
